std_msgs/Header header
float32[] optimal_policy 
float32[] action
float32 reward